doc/setup.md

```diff
@@ -38,6 +38,7 @@ class AppKernel extends Kernel
         $bundles = [
             ...
             //dependencies
+            new Symfony\Bundle\AsseticBundle\AsseticBundle(),
             new Troopers\AsseticInjectorBundle\TroopersAsseticInjectorBundle(),
             new Troopers\AlertifyBundle\TroopersAlertifyBundle(),
             new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```

Comment:
I was referencing this line: `new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),` ;)